app/components/layout/sidebar/multi_level_menu_component.html.erb

```diff
@@ -0,0 +1,71 @@
+<div data-controller="collapsible">
+  <button
+    type="button"
+    class="
+      flex
+      w-full
+      items-center
+      mt-1
+      p-3
+      transition-colors
+      ease-in-out
+      delay-75
+      hover:bg-slate-200
+      dark:text-slate-200
+      dark:hover:bg-slate-700
+    "
+    aria-controls="<%= "multi-level-menu-#{title}"%>"
```

Comment:
**suggestion:** Maybe `snakecase` the title here and below so that it is one work and  making it a proper id

Reply:
Updated in 006d96d5658b2d4f6b9eda2537fa32e2eea59dbe